.bumpversion.cfg

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 0.16.1
```

Comment:
risk: there is an OSS project (apioc) on the public npm with that name.
to avoid collisions, should we bump to a big version number or change the project name?

Reply:
we pull from our private repo first. But in that case even better we scope the package with @linkurious